README.md

```diff
@@ -138,6 +140,16 @@ If for some reason it's erroring out as `ERROR clone (Operation not permitted)`,
 your user is a member of the required `group` and that `xbps-uchroot(1)` utility has the
 proper permissions and owner/group as explained above.
 
+#### bwrap(1)
+
+bubblewrap, setuid implementation of a subset of user namespaces
+for unprivileged users. Requires no other permissions.
+See <https://github.com/containers/bubblewrap>.
```

Comment:
Our bwrap isn't suid, it uses user namespaces as well. I think something like "sandbox tool that can use user namespaces or be suid" would be more adequate.

Reply:
done